Correct the way of joining sessions (let's take info about only the last month)

sphinks opened the pull request:
For issue: https://github.com/fishtown-analytics/snowplow/issues/68

sphinks committed:
Correct the way of joining sessions (let's take info about only the last month)

drewbanin commented:
Thanks for making this PR @sphinks! I need to think a lot harder about the way this package should change. I'm very open to change like this, but I'd like to discuss the implementation further in #68 first!

I won't be able to get back to this in the very near term, but we are super interested in speeding up this package, and a mechanism like you've provided here is a very good way to do that! If this works for you, I'd recommend running your package from your fork for the time being. Just wanted to give you a heads up!

sphinks commented:
@drewbanin Thanks, I will continue trying to speed up the package.

sphinks committed:
Improve performance of page_view model
- Add sort key for web_events model
- Replace 'similar to' with more faster 'like'
- Search for session events only 1 day back instead of scanning the whole table

sphinks commented:
Closing PR as it not valid anymore. There will be another PR that allows speed up model.
